app/models/claims_table.rb

```diff
@@ -0,0 +1,4 @@
+class ClaimsTable < ActiveRecord::Base
+  self.abstract_class = true
+  establish_connection "claims_#{Rails.env}".to_sym
```

Comment:
Can't decide if I want to write unit tests for all this. If I do, then I'd setup a `claims_test`.

And then I'll be putting this under `claims_production` in production.